atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/internalschema/TransactionSchemaInstaller.java

```diff
@@ -34,7 +34,7 @@ public final class TransactionSchemaInstaller implements AutoCloseable {
     private static final SafeLogger log = SafeLoggerFactory.get(TransactionSchemaInstaller.class);
 
     @VisibleForTesting
-    static final Duration POLLING_INTERVAL = Duration.ofDays(3);
+    static final Duration POLLING_INTERVAL = Duration.ofMinutes(3);
```

Comment:
Is this just to speed up rollout?

Reply:
This was actually from the first patch on this branch, when we were seeing that stacks would refuse to downgrade from 3 to 2 because of some race conditions in startup.